src/Numerics/DGMethods/NumericalFluxes.jl

```diff
@@ -441,7 +441,7 @@ function numerical_flux_divergence!(
     grad⁺::Grad{GL},
 ) where {GL}
     parent(div_penalty) .=
-        (parent(grad⁺) .- parent(grad⁻))' * (normal_vector / 2)
+        (parent(grad⁺) .+ parent(grad⁻))' * (normal_vector / 2)
```

Comment:
Just curious/poking through things, why is this sign changing?

Reply:
I sneaked in a small functional change while rewriting hyperdiffusion changes to allow different polynomial orders. Now all divergence kernels are always done using weak form, and gradients are always done using strong form. This simplifies things a bit and allows symmetric treatment of hyperdiffusion on the sphere (what we had before was correct, but not symmetric). As an added bonus hyperdiffusion can now run with high polynomial orders on the GPU.